api/services/control/control.proto

```diff
@@ -66,19 +66,6 @@ message SolveRequest {
 }
 
 message CacheOptions {
-	// ExportRefDeprecated is deprecated in favor or the new Exports since BuildKit v0.4.0.
-	// When ExportRefDeprecated is set, the solver appends
-	// {.Type = "registry", .Attrs = ExportAttrs.add("ref", ExportRef)}
-	// to Exports for compatibility. (planned to be removed)
-	string ExportRefDeprecated = 1;
-	// ImportRefsDeprecated is deprecated in favor or the new Imports since BuildKit v0.4.0.
-	// When ImportRefsDeprecated is set, the solver appends
-	// {.Type = "registry", .Attrs = {"ref": importRef}}
-	// for each of the ImportRefs entry to Imports for compatibility. (planned to be removed)
-	repeated string ImportRefsDeprecated = 2;
-	// ExportAttrsDeprecated is deprecated since BuildKit v0.4.0.
-	// See the description of ExportRefDeprecated.
-	map<string, string> ExportAttrsDeprecated = 3;
```

Comment:
Can we add a comment: `ExportRefDeprecated =1, ImportRefsDeprecated = 2, ExportAttrsDeprecated = 3 were removed in BuildKit v0.6.0 and ignored`

Reply:
sure, will do.